packages/hoprd/src/commands/closeChannel.ts

```diff
@@ -31,13 +31,15 @@ export default class CloseChannel extends AbstractCommand {
 
     try {
       const { status, receipt } = await this.node.closeChannel(peerId)
+      const smartContractInfo = await this.node.smartContractInfo()
+      const channelClosureTime = Math.ceil(smartContractInfo.channelClosureTime / 60)
```

Comment:
I would suggest adding a unit specifier to this field's name. Also `60` could use some units.

Reply:
see cc4314c2b77c74d35701afbb81b9693bd9ef23fd